ADCL/src/communication/ADCL_allreduce_recursivedoubling.c

```diff
@@ -133,7 +133,8 @@ void ADCL_allreduce_recursivedoubling ( ADCL_request_t *req )
    tmprecv = (char*) rbuf;
 
    /* Determine nearest power of two less than or equal to size */
-   for (adjsize = 0x1; adjsize <= size; adjsize <<= 1); adjsize = adjsize >> 1;
+   for (adjsize = 0x1; adjsize <= size; adjsize <<= 1)
+       adjsize = adjsize >> 1;
```

Comment:
@edgargabriel Looking at this fix in the light of day, I think this one particular fix was wrong.  It should probably be:

```c
for (adjsize = 0x1; adjsize <= size; adjsize <<= 1)
    continue;   
adjsize = adjsize >> 1;
```

It's cases like this why we have the "always use {} for all blocks, even 1-line blocks" style rule in Open MPI.  :smile: